products/shopware.md

```diff
@@ -0,0 +1,184 @@
+---
+title: Shopware
+category: server-app
+tags: php-runtime
+iconSlug: shopware
+permalink: /shopware
+# alternate_urls: /
+versionCommand: php bin/console --version
+# releasePolicyLink: https://shopware.com/
+changelogTemplate: "https://github.com/shopware/shopware/blob/{{'__LATEST__'}}/CHANGELOG.md"
+releaseDateColumn: true
+activeSupportColumn: Bug fix maintenance
+eolColumn: Security maintenance
+# extendedSupportColumn: Shopware end of software support
+
+auto:
+-   git: https://github.com/shopware/platform.git
+
+# EOL dates on https://
+# PHP requirements on https://docs.shopware.com/en/shopware-6-en/first-steps/system-requirements
+releases:
+
+-   releaseCycle: "6.5"
+    releaseDate: 2023-05-03
+    support: true
+    eol: false
+    extendedSupport: true
+    latest: "6.5.6.0"
+    latestReleaseDate: 2023-10-04
+    link: https://github.com/shopware/shopware/blob/v6.5.6.0/CHANGELOG.md
+        
+-   releaseCycle: "6.4"
+    releaseDate: 2021-05-04
+    support: false
+    eol: false
+    extendedSupport: true
+    latest: "6.4.20.2"
+    latestReleaseDate: 2023-04-17
+    link: https://github.com/shopware/shopware/blob/v6.4.20.2/CHANGELOG.md
+
+-   releaseCycle: "6.3"
+    releaseDate: 2020-08-03
+    support: false
+    eol: false
+    extendedSupport: true
+    supportedPhpVersions: 7.2
+    latest: "6.3.5.4"
+    latestReleaseDate: 2021-04-19
+    link: https://github.com/shopware/shopware/blob/v6.3.5.4/CHANGELOG.md
+
+-   releaseCycle: "6.2"
+    releaseDate: 2020-05-19
+    support: false
+    eol: false
+    extendedSupport: true
+    supportedPhpVersions: 7.2
+    latest: "6.2.3"
+    latestReleaseDate: 2020-07-13
+    link: https://github.com/shopware/shopware/blob/v6.2.3/CHANGELOG-6.2.md
+
+-   releaseCycle: "6.1"
+    releaseDate: 2020-01-27
+    support: false
+    eol: false
+    extendedSupport: true
+    supportedPhpVersions: 7.2
+    latest: "6.1.6"
+    latestReleaseDate: 2020-05-04
+    link: https://github.com/shopware/shopware/blob/v6.1.6/CHANGELOG-6.1.md
+    
+-   releaseCycle: "5.7"
+    releaseDate: 2021-06-14
+    support: false
+    eol: true
+    extendedSupport: true
```

Comment:
https://safefive.de/en/faq-2/ says:

> You can count on our service for 5 years starting August 2024. We will regularly inform you about planned extensions of our offer.

```suggestion
    extendedSupport: 2029-08-01
```

Few questions:

1. It's unclear whether safefive will offer extended support for all 5.x versions or just the 5.7 series.
2. The safefive website mentions that Shopware 5 EOL is Aug **2024**, so should we have `eol: 2024-08-01` instead?